repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasAbstractDefStoreV2.java

```diff
@@ -58,6 +59,22 @@
     private static final Pattern INTERNAL_NAME_PATTERN = Pattern.compile(INTERNAL_NAME_REGEX);
 
     public static final String ALLOW_RESERVED_KEYWORDS = "atlas.types.allowReservedKeywords";
+    public static final List<String> KEYWORDS_INVALID_FOR_TYPE_CREATION = new ArrayList() {
+        {
+            add("boolean");
+            add("byte");
+            add("short");
+            add("int");
+            add("long");
+            add("float");
+            add("double");
+            add("biginteger");
+            add("bigdecimal");
+            add("string");
+            add("date");
+            add("objectid");
+        }
+    };
```

Comment:
Don't create static list but try to leverage builtInTypes defined inside typeRegistry
![image](https://user-images.githubusercontent.com/59254445/219013630-039d4ec6-0557-4dcb-bde4-13ddc5d3007c.png)